SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -45,7 +45,7 @@ Name:           ca-certificates
 # When updating, "Epoch, "Version", AND "Release" tags must be updated in the "prebuilt-ca-certificates*" packages as well.
 Epoch:          1
 Version:        2.0.0
-Release:        10%{?dist}
+Release:        11%{?dist}
```

Comment:
Would this change have a possibility to break any of packages that might not be aware of the breaking change where `blacklist` -> `blocklist` ?

Reply:
Good catch! I've added symbolic links now instead of removing directories.